src/platform/windows/display_base.cpp

```diff
@@ -510,6 +413,22 @@ namespace platf::dxgi {
     return false;
   }
 
+  /**
+   * @brief Hook for NtGdiDdDDIGetCachedHybridQueryValue() from win32u.dll.
+   * @param gpuPreference On output, the preference value.
```

Comment:
I'm not sure I understand this description. Is it meant to be like `An output for the preference value.`?

Reply:
Will clarify.